Common/Data/Custom/SmartInsider/SmartInsiderExecutionType.cs

```diff
@@ -41,6 +41,11 @@ public enum SmartInsiderExecution
         /// Under a specific agreement between the issuer and shareholder
         /// </summary>
         [EnumMember(Value = "Off Market Agreement")]
-        OffMarket
+        OffMarket,
+
+        /// <summary>
+        /// Field is not in this enum
+        /// </summary>
+        Error
```

Comment:
I would expect us to be adding new enum values in this PR too, the ones that triggered this error?

Reply:
The value provided to us is non-standard and the value it represents can not be mapped to an expected value, nor can it be inferred.

The value in question is an error, and should be parsed as such.